### ISSUE TITLE
Elytra false positive

### ISSUE THREAD

**Issue by atuna2** [Updated-NoCheatPlus/NoCheatPlus#132]
### Full output of /ncp version
їVersion informationЂ
Server:
git-Airplane-55 ~MC: 1.17.1~
Detected: 1.17.1
NoCheatPlus:
Plugin: 3.17-SNAPSHOT-Updated-b101
MCAccess: 1.13-1.17|? / Bukkit-API
Features:
blocks: BlocksMC1_4, BlocksMC1_5, BlocksMC1_6_1, BlocksMC1_7_2, BlocksMC1_8, BlocksMC1_9, BlocksMC1_10, BlocksMC1_11, BlocksMC1_12, BlocksMC1_13, BlocksMC1_14, BlocksMC1_15, BlocksMC1_16, BlocksMC1_17, MCAccessBukkitModern
checks: FastConsume, Gutenberg, HotFixFallingBlockPortalEnter, FlyingFrequency, KeepAliveFrequency, WrongTurn
defaults: pvpKnockBackVelocity
packet-listeners: MovingFlying, OutgoingPosition, KeepAliveAdapter, SoundDistance, WrongTurnAdapter, NoSlow, Fight
Hooks:
AllViolations~NCP~ 1.0, ViolationFrequency~NCP~ 1.0
їRelated PluginsЂ
ProtocolLib v4.7.0

### Describe the issue
Random stopping on ground when player trying to take off on elytra with firework_rocket.
Random stopping in air when player trying to boost with firework_rocket.

### How to reproduce the issue
Just try to take off with firework rocket or try to fly few minutes.

### Extra links/Videos (Including debug logs)
https://www.youtube.com/watch?v=ijO9y-Q25Xk 
https://pastebin.com/xSbcQbJ1

### Any possible config options changed or plugins that may cause interference?
Default configs

**Comment by deciduous-buffalo** [comment]
+1, this doesn't happen every time but around 25% of the time when trying to boost with rockets in 1.17 it lags you in-place until the boost ends. Generally happens when you are close to the ground. Never was an issue in 1.16.5 version

**Comment by Lysandr0** [comment]
(Adding @SpacePuppeh 's log too, just in case.  https://paste.lucko.me/JTmLMgY4LR )

**Comment by molor** [comment]
Only this bug now prevents me from updating to 1.17 :c

**Comment by compieter-gh** [comment]
+1

**Comment by MrRazamataz** [comment]
Set strict elytra to false and see if it helps

**Comment by Lysandr0** [comment]
Does the issue still persist with build n.108? A fix was implemented by @xaw3ep.

**Comment by Lysandr0** [comment]
Closing in favour of #150
